src/types/adapter.d.ts

```diff
@@ -47,4 +47,18 @@ export type AdapterCreator = (
 	displayName: string,
 	chatClient: ChatClient,
 	logger: ACSAdapterLogger,
-	featureOption: IAdapterInitFeatureOption) => IChatAdapter;
\ No newline at end of file
+	featureOption: IAdapterInitFeatureOption) => IChatAdapter;
+
+export type OmnichannelGenericResponse = {
+	data : {} | undefined;
```

Comment:
When defining somthing as `{}` it can also mean that it could be a number, null, string. It behaves more like `any`.
But if we know it's an object with properties like `{ key: "value" }` it's better to use a record
```suggestion
	data? : Record<string, unknown>;
```

Reply:
the idea is to be a generic wrapper